tests/robot-tests/run_tests.py

```diff
@@ -186,13 +190,17 @@
     robotArgs += ["--removekeywords", "name:operatingsystem.environment variable should be set"]
     robotArgs += ['--removekeywords',
                   'name:common.user goes to url']  # To hide basic auth credentials
+
+if args.custom_env: 
+    load_dotenv(args.custom_env)
+
 else:
     load_dotenv('.env.' + args.env)
```

Comment:
Just realised that this should all be in an else after the `if args.ci:` above. Right now the script will try to load a .env file when running on the pipeline.
```
if args.ci:
  ....
else:
  if args.custom_env
    ...
  else
    ...
  assert
```